client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -87,6 +91,17 @@ const PluginsBrowserList = ( {
 		}
 	};
 
+	const spotlightOnClick = () => {
+		dispatch(
+			recordTracksEvent( 'calypso_marketplace_spotlight_click', {
```

Comment:
Do we also need to record impressions, to measure click-through rate? Like `calypso_banner_cta_impression`

Reply:
It is a good idea, I think we have to create a new task for that, what do you think?

Reply:
Sounds good!